src/bindings/python/CMakeLists.txt

```diff
@@ -173,17 +173,9 @@ if(ENABLE_WHEEL)
     add_subdirectory(wheel)
 endif()
 
-if(ENABLE_OV_CORE_UNIT_TESTS AND NOT DISABLE_PYBIND11)
-    set(LIBRARY_OUTPUT_DIRECTORY_BIN ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+if(ENABLE_TESTS AND NOT DISABLE_PYBIND11)
```

Comment:
I suppose we need to remove usage of `DISABLE_PYBIND11` everywhere

Reply:
I will relay on your suggestion about it, as I don't know the reason of introducing this option (is feature like or workaround some CI issues).
@jiwaszki can you share your opinion ?

Reply:
This option was created to disable pybind-based Python API because of issues with cross-compilation which were fixed by you in this PR https://github.com/openvinotoolkit/openvino/pull/12775/files :) 
I suppose we just forgot to remove the option in that PR ;)

Reply:
Done